webpack/assets/javascripts/react_app/components/notifications/NotificationDrawer.js

```diff
@@ -24,6 +24,7 @@ class NotificationDrawer extends Component {
 
   onChange(actionType) {
     switch (actionType) {
+      case ACTIONS.MARK_AS_READ:
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({ notifications: NotificationsStore.getNotifications() });
```

Comment:
"no-fallthrough" will error in case you forget to "return" or "break" on a case.

not in cases where multiple cases are handled the same, which is fine and even important and useful in redux.

take for example a loading spinner for the screen, that should show up when any of several actions are fired (LOGIN_REQUEST, FETCH_DATA_REQUEST).